code/html/index.html

```diff
@@ -1,16 +1,34 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
 
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width , initial-scale=1">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   <script src="../js/rcalc-loader.js"></script>
+
+  <!-- i18n libraries supporting localization -->
+  <script src="../js/lib/CLDRPluralRuleParser/CLDRPluralRuleParser.js"></script>
```

Comment:
Why we have loader for JS files?

Reply:
Perfect ... loader looks nice. Where it was problem?

Reply:
to be honest I have no clue what was causing not wanting to work. I added one line of code jsScript.async = false; to make sure the order of loading is preserved.